reference/artifacts-with-artifactsrewrite/types/kubernetes-object/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: single
+title:  "Kubernetes Object"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+Kubernetes Object artifacts are _running_, _deployed_ Kubernetes manifests.
+This is in contrast to something like a GitHub file that only contains the
+specification of a deployable Kubernetes manifest.
+
+## Kubernetes Object Artifact in the UI
+
+The pipeline UI exposes the following fields for the Kubernetes object artifact:
+
+<table>
+  <thead>
+    <tr>
+      <th>Field</th>
+      <th>Explanation</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>Account</strong></td>
+      <td>An HTTP artifact account.</td>
+    </tr>
+    <tr>
+      <td><strong>URL</strong></td>
+      <td>The fully-qualified URL from which the file can be read.</td>
+    </tr>
+  </tbody>
+</table>
+
+### In a Trigger
```

Comment:
I don't think there's actually any way to use a Kubernetes artifact in a trigger or in a pipeline stage at the moment; these are currently only ever emitted by stages to show what was deployed.  While I could imagine some uses cases where we might want to accept them as input to stages, this functionality doesn't exist so we should probably just remove the "In a Trigger" and "In a Pipeline Stage" sections from this type.

Reply:
👍 Thanks!